.github/workflows/actionlint.yml

```diff
@@ -36,3 +36,15 @@ jobs:
           fail_on_error: true
           filter_mode: nofilter
           level: error
+  
+  check-runs-on:
+    needs: [ actionlint ]
+    runs-on: ubuntu-22.04
+    steps:
+      - run: |
+          ls -a
+          if grep -ER '^\s*runs-on:.*-latest' .github/workflows
+          then
+            echo "Use LTS versions to run on"
+            exit 1
+          fi
```

Comment:
Haha, nice! My question was if `actionlint` allows some runners to be disallowed. But I like the approach!
If it's not possible to do within the `actionlint`, then let's proceed with a bash script:
- Let's move the step to `actionlint` job (there's no need to have a separate job for this)
- And let's make the error message a bit more GitHub friendly (to allow not to check logs): https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error

Reply:
Yes, I am reading the docs now.